packages/block-editor/src/components/inserter-list-item/index.js

```diff
@@ -65,4 +66,4 @@ function InserterListItem( {
 	);
 }
 
-export default InserterListItem;
+export default memo( InserterListItem );
```

Comment:
I can see that being useful as we don't want to "rerender all items" when just one of them changes.